t/admin/schema.t

```diff
@@ -40,7 +40,7 @@ qr/"plugins":\{"type":"object"}/
 --- request
 GET /apisix/admin/schema/service
 --- response_body eval
-qr/"required":\["upstream"\]/
+qr/"required".*additionalProperties/
```

Comment:
why change this line?

Reply:
Because after removing the "required" of service in the schema_def.lua file, it will not match `qr/"required":\["upstream"\]/ `.

Reply:
this way is not easy to confirm if we removed `anyOf`, we should check it in Lua land